crates/sui-json-rpc/src/read_api.rs

```diff
@@ -230,12 +231,13 @@ impl RpcFullNodeReadApiServer for FullNodeApi {
         let (txn_data, txn_digest) = get_transaction_data_and_digest(tx_bytes)?;
         Ok(self
             .state
-            .dev_inspect_transaction(txn_data, txn_digest)
+            .dev_inspect_transaction(txn_data, txn_digest, 0)
             .await?)
     }
 
     async fn dev_inspect_move_call(
         &self,
+        epoch: EpochId,
```

Comment:
ditto, I would either remove this or add it as an optional as the last parameter